test/autoschedulers/mullapudi2016/reorder.cpp

```diff
@@ -161,51 +164,48 @@ int main(int argc, char **argv) {
 
     load_plugin(argv[1]);
 
-    const double slowdown_factor = 6.0;
-
     {
-        std::cout << "Test 1:\n";
         double manual_time = run_test_1(false);
         double auto_time = run_test_1(true);
 
-        std::cout << "======================\n"
-                  << "Manual time: " << manual_time << "ms\n"
-                  << "Auto time: " << auto_time << "ms\n"
-                  << "======================\n";
-
-        if (auto_time > manual_time * slowdown_factor) {
-            fprintf(stderr, "Warning: Auto-scheduler is much much slower than it should be.\n");
+        const double slowdown_factor = 15.0;  // TODO: whoa
```

Comment:
lol, I wonder which commit slowed this down by ~3x

Reply:
We never bothered checking the results before, we just logged to stderr

Reply:
Presumably it wasn't worse than 6x slower when it was written, though?